index.html

```diff
@@ -0,0 +1,156 @@
+                lovers, with the aim to expressing a way to celebrating glorious country heritage, culture and
+                traditions. This fest is meant to rejoice our special moments and emotions in our lives with our loved
+                ones, be it as it may, we're all country boys and girls.</p>
+            <h3 class="date-format">2022.11.11(FRI) ~ 12(SAT)</h3>
+            <p class="pragraph2">@ Ryman Auditorium, Nashville, Tennessee and more</p>
+
+        </div>
+
+    </header>
+
+    <div class="event-cards">
+        <h1 class="E-Program">Main Program</h1>
+        <div class="hr"></div>
+        <div class="card-modal">
+            <div class="card">
+                <img class="icon" src="images/lecturer-icon.svg" alt="">
+                <h2 class="card-title">Old Country and Western</h2>
+                <p class="card-desc">
+                    Listen to some good old, vintage, and classic country musicians
+                    play
+                </p>
+            </div>
+            <div class="card">
+                <img class="icon" src="images/business-presentation-icon.svg" alt="">
+                <h2 class="card-title">Tennessee Whiskey</h2>
+                <p class="card-desc">
+                    You have to determine how much whiskey you can take.
+                </p>
+
+            </div>
+            <div class="card">
+                <img class="icon" src="images/discuss-svgrepo-icon.svg" alt="">
+                <h2 class="card-title">Good Modern Country</h2>
+                <p class="card-desc">
+                    See Modern Country boys sing the souls of country music live.
+                </p>
+
+            </div>
+            <div class="card">
+                <img class="icon" src="images/connect-svgrepo-icon.svg" alt="">
+                <h2 class="card-title">Nice Hunting and Fishing</h2>
+                <p class="card-desc">
+                    Some country talents will be selected among attendees.
+                </p>
+
+            </div>
+            <div class="card">
+                <img class="icon" src="images/lecturer-icon.svg" alt="">
+                <h2 class="card-title">Long Neck, Ice Cold</h2>
+                <p class="card-desc">
+                    A time to chat and socialize while gulping that crisp taste.
+                </p>
+
+            </div>
+            <div class="card">
+                <img class="icon" src="images/business-presentation-icon.svg" alt="">
+                <h2 class="card-title">Good ol' Truck Mountain</h2>
+                <p class="card-desc">
+                    A game of who gets to sing 'My good ol' truck' after the festival.
+                </p>
+            </div>
+
+        </div>
+        <p class="btn-program">
+            <a href="">SEE THE WHOLE PROGRAM</a>
+        </p>
+        <div class="hr"></div>
+    </div>
```

Comment:
- [ ] Please consider wrapping this section using a `<section>` element rather than a `<div>`. This is best practice for HTML semantics.